pipeline-ui/src/components/IncreaseCounter.tsx

```diff
@@ -0,0 +1,42 @@
+import { Component } from 'react';
+import { types, WebMode } from '@algo-builder/web';
+import { NetworkType } from '../types';
+const { Button, Pipeline } = require('pipeline-ui');
+declare var AlgoSigner: any; // eslint-disable-line
+
+interface IncreaseCounterProps {
+	addr: string;
+	appId: number;
+}
+
+class IncreaseCounter extends Component<IncreaseCounterProps, {}> {
+	appCall = () => {
+		const networkType = Pipeline.main
+			? NetworkType.MAIN_NET
+			: NetworkType.TEST_NET;
```

Comment:
In the UI screenshot, I can only see the main net config

Reply:
Unchecking the checkbox will trated as TestNet. This component is from pipeline.

Reply:
This is a little weird for me, but thanks for informing